tests/scripts/analyze_outcomes.py

```diff
@@ -120,9 +133,10 @@ def analyze_driver_vs_reference(results: Results, outcomes,
         if test_suite in ignored_suites or full_test_suite in ignored_suites:
             continue
         # Skip ignored test cases inside test suites
-        if ((full_test_suite in ignored_test) and
-                (test_string in ignored_test[full_test_suite])):
-            continue
+        if full_test_suite in ignored_tests:
+            for str_or_re in ignored_tests[full_test_suite]:
```

Comment:
I wonder if it would make more sense to ignore regexes over a whole `.function` file? I'd generally expect ignore decisions to be somewhat orthogonal to how test cases are split between data files.

Reply:
Good idea. I think this might help when we do parity-testing in a build that has only cipher-light: we'll want to ignore CBC etc in all `.data` files regardless of whether it's aes, aria, etc.